src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXToZHigh.td

```diff
@@ -201,7 +212,23 @@ def replaceONNXDivPattern : Pat<
   (ONNXDivOp $x, $y),
   (ZHighUnstickOp (ZHighDivOp (ZHighStickOp:$s_x $x, (NoneLayoutAttr)),
                               (ZHighStickOp $y, (NoneLayoutAttr)),
-                              (returnType $s_x)))
+                              (returnType $s_x))),
+  [], [],
+  (addBenefit 0)
+>;
+
+def replaceONNXDivBroadcastPattern : Pat<
+  (ONNXDivOp (either $x, $y)),
+  (ZHighUnstickOp
+    (ZHighDivOp
+      (ZHighStickOp:$s_x $x, (NoneLayoutAttr)),
+      (ZHighStickifiedConstantOfShapeOp
```

Comment:
Any chance to enable ZHighStickify Op to handle broadcast?

Reply:
What is the dynShape approach?

Reply:
You pass the dynShape to create the broadcast constant.  Combine stickily and tensor expansion into one op.